configuration/amd64/docker-compose.yml

```diff
@@ -34,7 +34,10 @@ services:
     depends_on: 
       - php
     environment:
-      - MYSQL_PASSWORD=/run/secrets/mysql_password
+      - MYSQL_DATABASE=IVP
```

Comment:
> Had some time this morning to test your branch. Found some issues. The reason CARMA-Streets did not appear in UI is because you didn't add it to the list of plugins installed in the container/service.sh file. I also made the v2xhub container only depend on db and the php container depend on mysql and v2xhub. I think this is the correct way to set this up but it should work with the normal configuration to I think.

@paulbourelly999 Thanks a lot. It is updated.